.travis.yml

```diff
@@ -7,6 +7,7 @@ go:
 
 env:
   - GO111MODULE=on
```

Comment:
Change it to be only one line or it will create several tests:

```
- GO111MODULE=on GOPROXY=https://proxy.golang.org
```

Reply:
done